util/AppLoggingHandler.go

```diff
@@ -0,0 +1,58 @@
+package util
+
+import (
+	"io"
+	"log"
+	"os"
+	"os/exec"
+	"os/signal"
+	"syscall"
+)
+
+// SpawnProcessWithLogging
+// This method handles the logic for maintaining a local log file for log archival and handles SIGTERM propagation.
+// The subprocess spawned will bypass this function and execute the main logic
+func SpawnProcessWithLogging() {
+
+	// Create an in-memory pipe
+	pr, pw := io.Pipe()
```

Comment:
will there be any impact because of large log file ??

Reply:
the rate of data  in and out depends on the amount logs being emitted which is reasonable for our use case, therefore there shouldn't be any impact